lib/expression/v2/callbacks/standard.ex

```diff
@@ -0,0 +1,1311 @@
+defmodule Expression.V2.Callbacks.Standard do
```

Comment:
copied from Expression.Callbacks.Standard and updated for the new setup, removed a lot of eval! and eval_args! calls